Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Driver.hs

```diff
@@ -517,7 +517,8 @@ data VerificationStatus = PENDING | VALID | INVALID
 
 data RCStatusReq = RCStatusReq
   { rcNo :: Text,
-    isActivate :: Bool
+    isActivate :: Bool,
+    downgradePreference :: Maybe Bool
```

Comment:
Can we take 3 flags here instead? SUV might want to downgrade to Sedan but not Hatchback

Reply:
Sure, I have made the required changes